### PR TITLE
feature(TPG>=5.39)!: add resource_tags

### PR DESCRIPTION
**Pull request by lord-skinner**

### CODE REVIEW

**Comment by lord-skinner** [comment]
@davenportjw @shanecglass The CI error failing is `Needs /gcbrun from a collaborator`, but I'm not sure how to proceed with this information. 

Can you let us know how to proceed? 

cc:@rohilla-anuj

**Review by bharathkkb — APPROVED** [review]
Thanks for the PR! Could you also bump the min provider version [1] to > 5.39. Looks like thats when this field was added per https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#5390-july-29th-2024

[1]: https://github.com/terraform-google-modules/terraform-google-bigquery/blob/0fe8ab60d7291a2260cd460d55cdcca7fc815a0d/versions.tf#L23

**Comment by rohilla-anuj** [comment]
@bharathkkb Thanks for looking into the PR.
Made the change as per your suggestion.

**Review by bharathkkb — COMMENTED** [review]
A few lint errors are reported

- Run 'make docker_generate_docs' command and commit the generated readme update.
- Checking for trailing whitespace ./README.md:39

**Comment by rohilla-anuj** [comment]
Fixed the Lint error.
@bharathkkb can you approve /gcbrun again.

File: versions.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 5.3, < 7"
+      version = "> 5.39, < 7"
```
Comment (apeabody) [inline versions.tf:23]: Looks like this was added **in** 5.39, so >= should be fine.

```suggestion
      version = ">= 5.39, < 7"
```

**Comment by apeabody** [comment]
Thanks @lord-skinner - Can you please rebase this change (its not enabled for maintainers), otherwise it LGTM.

**Comment by apeabody** [comment]
> Thanks @lord-skinner - Can you please rebase this change (its not enabled for maintainers), otherwise it LGTM.

Thanks for the contribution @lord-skinner!

**Pull request merged**